dev/custom-pages/contact.html

```diff
@@ -0,0 +1,83 @@
+<div class="row">
+  <div class="col-sm-6">
+    <h2 class="h1 heading-slab-serif">We look forward to hearing from you.</h2>
+    <p>
+      But if you write us, know that under Florida law email addresses are public records. If you don’t want your email address released in response to a public records request, don’t send an email. Instead, contact UCF by phone or through the mail.
+    </p>
+  </div>
+  <div class="col-sm-6 col-md-5 col-md-offset-1">
+    <div class="margin-h1 well">
+      <div class="row">
+        <div class="hidden-xs col-sm-3 text-center">
+          <span class="fa fa-map-marker ucf-gold-dark fa-3x margin-h4" aria-hidden="true"></span>
+        </div>
+        <div class="col-sm-9">
+          <h3 class="h4 heading-sans-serif">Physical Address:</h3>
+          <address>
+            University of Central Florida<br>
+            4000 Central Florida Blvd.<br>
+            Orlando, FL 32816
+          </address>
+        </div>
+      </div>
+      <div class="row">
+        <div class="hidden-xs col-sm-3 text-center">
+          <span class="fa fa-envelope ucf-gold-dark fa-2x margin-h4" aria-hidden="true"></span>
+        </div>
+        <div class="col-sm-9">
+          <h3 class="h4 heading-sans-serif">Mailing Address:</h3>
+          <address>
+            University of Central Florida<br>
+            P.O. Box 160000<br>
+            Orlando, FL 32816
+          </address>
+        </div>
+      </div>
+      <div class="row">
+        <div class="hidden-xs col-sm-3 text-center">
+          <span class="fa fa-mobile-phone ucf-gold-dark fa-3x margin-h4" aria-hidden="true"></span>
+        </div>
+        <div class="col-sm-9">
+          <h3 class="h4 heading-sans-serif">Main Campus Phone:</h3>
+          <a href="tel:4078232000">(407) 823-2000</a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+</div></div></div>
+
+<aside class="callout callout-gold margin-h2">
+  <div class="container icon-links">
+    <div class="icon-link icon-link-dark">
+      <a href="http://www.ucf.edu/azindex/" target="_blank">
```

Comment:
I ended up not using relative urls due to our lack of a base tag in the document head for this theme; not having one will make relative links here render differently depending on how WordPress is installed, and whether or not the main site is the primary site on the given network. I'm going to add base tag addition as a backlog task for now (I hesitate to add one right away because I'm not 100% sure how it'll impact other links throughout the site), but I've updated these URLs to at least point to https.